crates/biome_test_utils/src/lib.rs

```diff
@@ -505,3 +506,43 @@ pub fn validate_eof_token<L: Language>(syntax: SyntaxNode<L>) {
         "the EOF token may not contain any data except trailing whitespace"
     );
 }
+
+/// Checks whether valid snapshot file contains
+/// `/* should not generate diagnostics */`" comment
+pub fn assert_valid_snapshot_contains_no_diagnostic_comment(
+    snapshot_path: &Utf8Path,
+    snapshot_code: &str,
+) {
+    let should_contain_comment = match snapshot_path.extension().unwrap_or_default() {
+        // Excluded files types which cannot contain comment in the source code
+        "snap" | "json" | "jsonc" | "svelte" | "vue" | "astro" | "html" => false,
```

Comment:
I felt like opt-out is safer in catching new file formats.

Svelte/vue/astro likely can be removed in the future, but I did not want to modify existing cases snapshots rn.